pyvcloud/vcd/utils.py

```diff
@@ -229,7 +229,7 @@ def disk_to_dict(disk):
     result['name'] = disk.get('name')
     result['id'] = extract_id(disk.get('id'))
     result['status'] = disk.get('status')
-    result['size_MB'] = disk.get('size')
+    result['size_bytes'] = disk.get('size')
```

Comment:
Have you tested this change? Where all in the code are we using disk_to_dict?

Reply:
I am going to revert this change, however, I will have a jira task to track this issue, as the vcli provides the size information for a disk in bytes but this misleads it as size in MB.